CONTRIBUTING.md

```diff
@@ -220,10 +220,12 @@ nox -s watch_docs
 
 ## Releasing New `pipx` Versions
 
-To publish to PyPI simply create a "published" release on GitHub. This will trigger GitHub workflows that publishes:
+To release a new version, manually run the `bump-changelog` action under the *"Actions"* tab, passing it the version to
+be released. This will create a pull request updating the changelog for the upcoming version, with the `release-version` label. Attaching this label to any pull request of which the title follows the format `<Version>: Description` and merging it will trigger GitHub workflows that publish:
```

Comment:
This to me is a bit clearer.

```suggestion
To release a new version, manually run the `bump-changelog` action under the *"Actions"* tab, passing it the version to
be released. This will create a pull request updating the changelog for the upcoming version, with the `release-version`
label. Merging this PR will automatically trigger the release workflows.

Attaching this label to any pull request of which the title follows the format `<Version>: Description` and merging it will 
trigger the release workflows as well.

The release workflow consists of publishing:
```